src/Mollie/WC/Gateway/Giftcard.php

```diff
@@ -97,7 +97,10 @@ public function payment_fields()
 
 		// If only one gift card issuers is available, show it without a dropdown
 		if ( count( $issuers ) === 1 ) {
-			$html .= '<img src="' . $this->getIssuerIconUrl( $issuers[0]->id ) . '" style="vertical-align:middle" />';
+            $issuerImageSvg = $this->checkSvgIssuers($issuers);
+            if(isset($issuerImageSvg)){
```

Comment:
Here you do not need to check if the variable is set, you're interested in the value so you can simply check this way `$issuerImageSvg and $html .= '<img src="' . $issuerImageSvg . '" style="vertical-align:middle" />';`